counter/src/main/java/org/infinispan/counter/impl/listener/CounterManagerNotificationManager.java

```diff
@@ -267,14 +271,12 @@ public void topologyChanged(TopologyChangedEvent<?, ?> event) {
                  .forEach(TopologyChangeListener::topologyChanged);
```

Comment:
I just noticed this above uses `parallelStream`. We shouldn't be using this if possible as it will use the common pool. Is there a reason for this usage?

Reply:
no reason, changed to sequential stream